Clean up dependencies: Ubuntu, Conan, Conda.

arthurp opened the pull request:
Remove conan dependencies accidentally reintroduced.
Move dependency into enterprise conan.

tylershunt commented:
I get a bunch of "Chile aborted**Exception"s when I run this in my environment. For instance the first is for  run-small-betweennesscentrality-outer-1. Is that expected behavior? Or did I miss some config step?

ddn0 commented:
> I get a bunch of "Chile aborted**Exception"s when I run this in my environment. For instance the first is for run-small-betweennesscentrality-outer-1. Is that expected behavior? Or did I miss some config step?

If you are running this locally, you need to do `make input`. I'll add something to the `test/README.md` about this step.

tylershunt commented:
Awesome, thanks!

arthurp committed:
Clean up dependencies: Ubuntu, Conan, Conda.

Remove conan dependencies accidentally reintroduced.
Move dependency into enterprise conan.

ddn0 commented on config/conanfile.py:
```diff
@@ -12,17 +12,12 @@ class KatanaConan(ConanFile):
         "eigen/3.3.7",
         "fmt/6.2.1",
         "libcurl/7.74.0",
-        "lz4/1.9.3",
```
I guess the `arrow` conan package will add these anyway if `arrow:with_foo` is True?

arthurp replied:
Yes. You removed them. ;-) https://github.com/KatanaGraph/katana-enterprise/commit/a5810eeaedb8bcfb05f6c00a0fbcc4796124ce44#diff-ce048b1297f1f5f485c949613942256e74b195a6e05dc761d179a2857382e38b
